src/containers/App.js

```diff
@@ -65,11 +65,40 @@ export default class App extends React.Component {
     return pick(this.props, 'dispatch', 'currentUser', 'isMobile')
   }
 
+  setLeftNavRefreshInterval (community) {
```

Comment:
hmmmm --- can you integrate this into fetchLiveStatus instead, to minimize the number of request-response roundtrips?